packages/react-components/lib/doors/door-panel.spec.tsx

```diff
@@ -0,0 +1,43 @@
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { DoorPanel } from './door-panel';
+import { doorStates, makeDetailedDoors } from './test-utils.spec';
+
+function renderDoorPanel() {
+  const mockControlClickSubmit = jasmine.createSpy();
+  return render(
+    <DoorPanel
+      doors={makeDetailedDoors()}
+      doorStates={doorStates}
+      onDoorControlClick={mockControlClickSubmit}
```

Comment:
The spy is not accessible from the tests so theres probably no need to have them here. If onDoorControlClick is a required props, should probably make it optional.